README.md

```diff
@@ -296,22 +296,22 @@ python wpt make-hosts-file | Out-File $env:SystemRoot\System32\drivers\etc\hosts
 If you are behind a proxy, you also need to make sure the domains above are excluded
 from your proxy lookups.
 
-#### 5. Set `WPT_BROWSER_PATH`
+#### 5. Set `CHROME_BIN`
```

Comment:
It can be not only Chrome, but Edge ow any other Chromium browser.

Reply:
Changed to `BROWSER_BIN`